ui/DesignSystem/Component/Avatar.svelte

```diff
@@ -7,7 +7,7 @@
   export let title = null;
 
   export let variant = "user"; // user | project
-  export let size = "regular"; // regular | medium | big
+  export let size = "regular"; // regular | medium | big | xl
```

Comment:
I just saw that in `Title.svelte` the next step after `big` is `huge`. I think it would be nice to make it consistent.